### DISCUSSION TITLE
Feature Request: The ability to use mdm app config to set the sync server url.

### DISCUSSION THREAD

**Discussion by techmechanic1** [standardnotes/forum#2572] [Suggestions]
Feature Request: The ability to use mdm app config to set the sync server url.. and would be nice to be able to set the username too...similar to Nextcloud, https://github.com/nextcloud/ios/blob/master/MDM/AppConfig/specfile.xml

**Comment by JaspalSuri** [comment]
Hi @PhoneMechanic, can you elaborate on this a bit more? Are you building the mobile apps and want to set the default syncing server URL and your account's username?

**Comment by techmechanic1** [comment]
Hi @JaspalSuri, What I was talking about was the ability to set the Syncing Server Url using managed app configuration.. https://www.appconfig.org/ios/#appconfiguration . Our organization is planning on starting to use Standard Notes with our own Syncing Server. It would be a lot easier for our users to register/signin if we could set the Syncing Server Url from our MDM...This isnt as high priority but I'd like to be able to set the username via app config too...

**Comment by JaspalSuri** [comment]
Hi @PhoneMechanic, thanks for elaborating and thank you for the suggestion!
